src/DeepLinkModule/.settings/org.eclipse.jdt.core.prefs

```diff
@@ -0,0 +1,15 @@
+eclipse.preferences.version=1

```

Comment:
Is this file/folder supposed to be in there?

Reply:
No, removed it from repo.